test/spec/provider/camunda/element-templates/parts/CustomProps.json

```diff
@@ -591,7 +591,7 @@
       },
       {
         "value": "println end",
-        "type": "String",
+        "type": "Hidden",
```

Comment:
What is this? A simple template fix?

Reply:
The template was actually invalid. `camunda:executionListener` bindings can't have the property type `String`.

Cf. https://github.com/camunda/camunda-modeler/tree/develop/docs/element-templates#camundaexecutionlistener